### DISCUSSION TITLE
[Trader Idea] Buy 10 - Buy All

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1242] [Ideas]
I do not think it is "**_politically correct_**" to have to impose whole packages of resources when perhaps these are not needed.

Time to mix, folks!

![Schermata 2022-07-16 alle 17 59 02](https://user-images.githubusercontent.com/58088988/179362551-ddbdd3b2-30a3-47e1-9eb0-7599e6534855.png)

- Quatrefoil

**Comment by adamhannigan** [comment]
Thanks for the suggestion @Q4FOIL 

Unfortunately we haven't had the chance to build out 'partial orders' in the trading system. This is quite a complex case and can lead to a range of vulnerabilities.

**Reply by Q4FOIL** [reply]
Hello @adamhannigan.

I suppose you also need to review the contracts, in any case it is really a shame, my idea is that every player must have full freedom within the game, for example in MoM it is perfectly structured.

Could this be the final version? :P

- Quatrefoil ❤️
